fabulaws-config.yml

```diff
@@ -196,3 +195,66 @@
     - build-essential
     - stunnel4
     - pgbouncer
+
+  db_settings:
```

Comment:
How did this end up in here? Looks like a duplicate of PR #16, which appears to have been merged already.